From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 3 Jan 2025 09:34:51 +0100
Subject: dt-bindings: input: goodix,gt9916: Document GT9897 touchscreen

Goodix GT9897 is a touchscreen from Goodix using the Berlin-A protocol
instead of Berlin-D like GT9916.

Signed-off-by: Luca Weiss <luca.weiss@fairphone.com>
---
 Documentation/devicetree/bindings/input/touchscreen/goodix,gt9916.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/input/touchscreen/goodix,gt9916.yaml b/Documentation/devicetree/bindings/input/touchscreen/goodix,gt9916.yaml
index d90f045..c40d92b 100644
--- a/Documentation/devicetree/bindings/input/touchscreen/goodix,gt9916.yaml
+++ b/Documentation/devicetree/bindings/input/touchscreen/goodix,gt9916.yaml
@@ -19,6 +19,7 @@ allOf:
 properties:
   compatible:
     enum:
+      - goodix,gt9897
       - goodix,gt9916
 
   reg:
